src/agent/onefuzz-task/src/tasks/fuzz/generator.rs

```diff
@@ -239,95 +288,99 @@ mod tests {
+            let crashes_local = tempfile::tempdir().unwrap().path().into();
+            let tools_local = tempfile::tempdir().unwrap().path().into();
+            let config = Config {
+                generator_exe: String::from("{tools_dir}/radamsa"),
+                generator_options,
+                readonly_inputs: vec![SyncedDir {
+                    local_path: readonly_inputs_local,
+                    remote_path: Some(BlobContainerUrl::parse(
+                        Url::from_directory_path(inputs).unwrap(),
+                    )?),
+                }],
+                crashes: SyncedDir {
+                    local_path: crashes_local,
+                    remote_path: Some(BlobContainerUrl::parse(
+                        Url::from_directory_path(crashes).unwrap(),
+                    )?),
                 },
-                tags: Default::default(),
-                from_agent_to_task_endpoint: "/".to_string(),
-                from_task_to_agent_endpoint: "/".to_string(),
-            },
-        };
-        let task = GeneratorTask::new(config);
+                tools: Some(SyncedDir {
+                    local_path: tools_local,
+                    remote_path: Some(BlobContainerUrl::parse(
+                        Url::from_directory_path(radamsa_dir).unwrap(),
+                    )?),
+                }),
+                target_exe: Default::default(),
+                target_env: Default::default(),
+                target_options: Default::default(),
+                target_timeout: None,
+                check_asan_log: false,
+                check_debugger: false,
+                rename_output: false,
+                ensemble_sync_delay: None,
+                generator_env: HashMap::default(),
+                check_retry_count: 0,
+                common: CommonConfig {
+                    job_id: Default::default(),
+                    task_id: Default::default(),
+                    instance_id: Default::default(),
+                    heartbeat_queue: Default::default(),
+                    job_result_queue: Default::default(),
+                    instance_telemetry_key: Default::default(),
+                    microsoft_telemetry_key: Default::default(),
+                    logs: Default::default(),
+                    setup_dir: Default::default(),
+                    extra_setup_dir: Default::default(),
+                    extra_output: Default::default(),
+                    min_available_memory_mb: Default::default(),
+                    machine_identity: onefuzz::machine_id::MachineIdentity {
+                        machine_id: uuid::Uuid::new_v4(),
+                        machine_name: "test".to_string(),
+                        scaleset_name: None,
+                    },
+                    tags: Default::default(),
+                    from_agent_to_task_endpoint: "/".to_string(),
+                    from_task_to_agent_endpoint: "/".to_string(),
```

Comment:
If `CommonConfig` implements `Default` then this can all be replaced by `CommonConfig { some_field: X, ..CommonConfig::default() }`